### DISCUSSION TITLE
Copying Members from protected object filters

### DISCUSSION THREAD

**Discussion by yeahitsaustin** [codefori/vscode-ibmi#1656] [Ideas]
It would be helpful to be able to copy members from protected object browser/filters (with right-click > Copy Member). 

Of course there are workarounds like temporarily making the filter unprotected or creating a separate unprotected filter over the same library, but copying directly from a protected filter would save lots of clicks over time. :)

**Comment by worksofliam** [comment]
Why wouldn't you be able to? We don't block the copy option if it's protected.

**Reply by sebjulliand** [reply]
The Copy action isn't actually available in protected filters. I opened an issue and a PR for that 😉
